contracts/HubPool.sol

```diff
@@ -196,17 +196,21 @@ contract HubPool is HubPoolInterface, Testable, Lockable, MultiCaller, Ownable {
     }
 
     function setBond(IERC20 newBondToken, uint256 newBondAmount) public onlyOwner noActiveRequests {
+        // todo: check that the bond token is an approved collateral currency.
+        // todo: add the final fee to the bond amount by grabbing from the store.
```

Comment:
Done.

Reply:
Where is this done? Can you link to lines of code in a PR that address these todos? I think that'd be easier to track